src/__tests__/commands/organizations.test.ts

```diff
@@ -0,0 +1,121 @@
+import { jest } from '@jest/globals'
+
+import type { ArgumentsCamelCase, Argv } from 'yargs'
+
+import type {
+	OrganizationResponse,
+	OrganizationsEndpoint,
+	SmartThingsClient,
+} from '@smartthings/core-sdk'
+
+import type { APICommand, APICommandFlags } from '../../lib/command/api-command.js'
+import type { outputItemOrList, outputItemOrListBuilder } from '../../lib/command/listing-io.js'
```

Comment:
These aren't types, are they?

Reply:
They are! And so much more. But alas, here I only care about the types.